AllEvents.TicketManagement/src/Core/AllEvents.TicketManagement.Application/Features/Tickets/Handlers/GenerateTicketCommandHandler.cs

```diff
@@ -0,0 +1,120 @@
+﻿using AllEvents.TicketManagement.Application.Contracts;
+using AllEvents.TicketManagement.Application.Features.Tickets.Commands;
+using AllEvents.TicketManagement.Application.Models;
+using AllEvents.TicketManagement.Domain.Entities;
+using MediatR;
+using QRCoder;
+using System.Drawing.Imaging;
+using System.Security.Cryptography;
+using System.Text;
+
+namespace AllEvents.TicketManagement.Application.Features.Tickets.Handlers
+{
+    public class GenerateTicketCommandHandler : IRequestHandler<GenerateTicketCommand, TicketModel>
+    {
+        private readonly IEventRepository _eventRepository;
+        private readonly ITicketRepository _ticketRepository;
+        private readonly byte[] aesKey = Encoding.UTF8.GetBytes("A3C9F7E20B567891"); 
```

Comment:
@KKDinev20  do not use hardcoded string for the key 
it should be Environment Variable